src/transformers/generation_tf_utils.py

```diff
@@ -1924,7 +1924,7 @@ def _prepare_model_inputs(self, inputs: Optional[tf.Tensor] = None, bos_token_id
                     "you should either supply a context to complete as `input_ids` input "
                     "or a `bos_token_id` (integer >= 0) as a first token to start the generation."
                 )
-            return tf.fill((1, 1), bos_token_id, dtype=tf.int32)
+            return tf.cast(tf.fill((1, 1), bos_token_id), dtype=tf.int32)
```

Comment:
Interesting, why is that needed?

Reply:
It seems like TF 2.8.0 doesn't have a `dtype` input arg anymore: https://www.tensorflow.org/api_docs/python/tf/fill . Not sure why this worked before